go/obscuronode/enclave/enclaverunner/enclave_runner.go

```diff
@@ -26,6 +27,7 @@ func RunEnclave(config EnclaveConfig) {
 	if err := enclave.StartServer(config.Address, nodeAddress, txHandler, false, nil, nil); err != nil {
 		panic(err)
 	}
+	fmt.Println("Obscuro enclave service started.")
```

Comment:
Should we log instead ?

Reply:
I did this on purpose so there's some sign of life when things are pushed to the logs. Otherwise, you can think the process is blocked/dead, but you just haven't checked the logs.

Reply:
Fair enough 👍